fraud_detection/Dockerfile

```diff
@@ -0,0 +1,44 @@
+############################################################################
+##
+## Copyright (C) 2021 NVIDIA Corporation.  All rights reserved.
+##
+## NVIDIA Sample Code
+##
+## Please refer to the NVIDIA end user license agreement (EULA) associated
+## with this source code for terms and conditions that govern your use of
+## this software. Any use, reproduction, disclosure, or distribution of
+## this software and related documentation outside the terms of the EULA
+## is strictly prohibited.
+##
+############################################################################
+#
+# This Dockerfile contains the needed NVIDIA 
+# libraries, and plugins to enable the notebooks for FSI Synthetic Data Creation
+#
+# BUILD: docker build --network=host --file Dockerfile --tag fsi_fd:nvidia .
+# RUN: docker run --gpus all --rm -it -p 8888:8888 -p 8889:8889 -p 8787:8787 -p 8788:8788 fsi_fd:nvidia 
+# JUPYTER NOTEBOOK: /jupyter
+
+##############################################################
+
+FROM nvcr.io/nvidia/rapidsai/rapidsai:0.19-cuda11.2-runtime-ubuntu20.04
+
+SHELL ["conda", "run", "-n", "rapids", "/bin/bash", "-c"]
+
+# Additional python libs
+ADD requirements.txt /
+RUN  pip install -r /requirements.txt \
+     && apt-get update && apt-get upgrade -y\
+     && apt-get install -y p7zip-full
+
```

Comment:
Got following error when building the docker image

```
The conflict is caused by:
    The user requested torch==1.9.0
    torchvision 0.9.0 depends on torch==1.8.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict


ERROR: Cannot install -r /requirements.txt (line 5) and torch==1.9.0 because these package versions have conflicting dependencies.
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

Reply:
Removed torchvision and torchaudio dependencies.